internal/testing/cmp/cmp.go

```diff
@@ -0,0 +1,76 @@
+/*
+ Copyright 2021 - 2022 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cmp
+
+import (
+	"strings"
+
+	gocmp "github.com/google/go-cmp/cmp"
+	gotest "gotest.tools/v3/assert/cmp"
+	"sigs.k8s.io/yaml"
+)
+
+type Comparison = gotest.Comparison
+
+// Contains succeeds if item is in collection. The collection may be a string,
+// map, slice, or array. See [gotest.tools/v3/assert/cmp.Contains].
+func Contains(collection, item interface{}) Comparison {
+	cString, cStringOK := collection.(string)
+	iString, iStringOK := item.(string)
+
+	if cStringOK && iStringOK {
+		if strings.Contains(cString, "\n") || strings.Contains(iString, "\n") {
```

Comment:
Ah, so if this is a single line string, we just use the line 51 implementation of Contains, yeah?

Reply:
Right!